src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraDeploymentContext.kt

```diff
@@ -75,68 +78,88 @@ fun List<AuroraDeploymentContext>.createDeployCommand(
+    return this.map { (context, eitherErrorsOrFeatures) ->
+        // There was some errors when generating so we gather them up and return them and no resources
+        val errors = eitherErrorsOrFeatures.mapNotNull { it.first }
+        if (errors.isNotEmpty()) {
+            return@map ErrorsOrFeatures(context, Pair(errors, null))
+        }
 
-    val featureResources = eitherErrorsOrFeatures.mapNotNull { it.second }.flatten().toSet()
-    val names = featureResources.map { "${it.resource.kind}/${it.resource.metadata.name}" }
+        val featureResources = eitherErrorsOrFeatures.mapNotNull { it.second }.flatten().toSet()
+        val names = featureResources.map { "${it.resource.kind}/${it.resource.metadata.name}" }
 
-    val uniqueNames = names.toSet()
+        val uniqueNames = names.toSet()
 
-    val duplicatedNames = uniqueNames.filter { uniqueName ->
-        names.count { uniqueName == it } != 1
-    }
+        val duplicatedNames = uniqueNames.filter { uniqueName ->
+            names.count { uniqueName == it } != 1
+        }
 
-    // TODO: This failed Test this
-    if (duplicatedNames.isNotEmpty()) {
-        val namesString = duplicatedNames.joinToString(", ").lowercase()
-        val error: List<ContextErrors> =
-            listOf(
+        // TODO: This failed Test this
+        if (duplicatedNames.isNotEmpty()) {
+            val namesString = duplicatedNames.joinToString(", ").lowercase()
+            val error: List<ContextErrors> = listOf(
                 ContextCreationErrors(
-                    this.cmd,
+                    context.cmd,
                     listOf(RuntimeException("The following resources are generated more then once $namesString"))
                 )
             )
-        return error to featureResources
-    }
-
-    // TODO: Message:     Resource with duplicate kind/name=Service/activemq-3-native,Service/activemq-3-admin,Route/activemq-3,ImageStream/amq-classic-single,DeploymentConfig/activemq-3,Secret/activemq-3-cert,Route/activemq-3,ApplicationDeployment/activemq-3,ProjectRequest/paas-m89870,Namespace/paas-m89870,RoleBinding/admin
+            return@map ErrorsOrFeatures(context, Pair(error, featureResources))
+        }
 
-    // Mutation!
-    val modifyErrors = this.features.mapNotNull {
-        val context = this.featureContext[it.key] ?: emptyMap()
-        try {
-            it.key.modify(it.value, featureResources, context)
-            null
-        } catch (e: Throwable) {
-            if (e is ExceptionList) {
-                ContextCreationErrors(this.cmd, e.exceptions)
-            } else {
-                ContextCreationErrors(this.cmd, listOf(e))
+        // TODO: Message:     Resource with duplicate kind/name=Service/activemq-3-native,Service/activemq-3-admin,Route/activemq-3,ImageStream/amq-classic-single,DeploymentConfig/activemq-3,Secret/activemq-3-cert,Route/activemq-3,ApplicationDeployment/activemq-3,ProjectRequest/paas-m89870,Namespace/paas-m89870,RoleBinding/admin
```

Comment:
Ser du en grunn for å bevare denne TODOen frode?

Reply:
Nei, jeg vet ikke hva den prøver å si heller, og jeg har lyst til å fjerne den

Reply:
Jeg ville fjernet den. Den ser ut som rester etter en debug sessjon